apollo-runtime/src/main/java/com/apollographql/apollo/ApolloClient.java

```diff
@@ -383,6 +422,7 @@ Response cachedHttpResponse(String cacheKey) throws IOException {
         .useHttpGetMethodForQueries(useHttpGetMethodForQueries)
         .useHttpGetMethodForPersistedQueries(useHttpGetMethodForPersistedQueries)
         .writeToNormalizedCacheAsynchronously(writeToNormalizedCacheAsynchronously)
+        .batchPoller(batchConfig.getBatchingEnabled() && canBeBatched ? batchPoller : null)
```

Comment:
Nitpicking, the `batchConfig.getBatchingEnabled()` seems uneeded as `batchPoller = null` in that case
```suggestion
        .batchPoller(canBeBatched ? batchPoller : null)
```

Reply:
good call